src/Elmish.WPF/WpfProgram.fs

```diff
@@ -88,7 +88,7 @@ module WpfProgram =
           element.DataContext <- vm
           viewModel <- Some vm
       | Some vm ->
-          vm.UpdateModel model
+          IViewModel.updateModel (vm, model)
```

Comment:
Can lines like this remain unchanged?

Reply:
I'm not sure exactly how it affects the inferred type generics. I think I would have to add a type annotation similar to what is in `IViewModel.updateModel`

Reply:
Done

Reply:
@TysonMN actually I figured out that the original implementation didn't have redundant `UpdateModel` and `CurrentModel` properties on it. So now it's actually *not* possible for it to remain unchanged.

Reply:
Its ok.  I sort of prefer the original syntax in this case (because it is shorter), but looking around the rest of the PR, I see that basically all other changes were an improvement (by being shorter).